docs/reference/ilm/index-rollover.asciidoc

```diff
@@ -12,7 +12,20 @@ Using rolling indices enables you to:
 * Shift older, less frequently accessed data to less expensive _cold_ nodes,
 * Delete data according to your retention policies by removing entire indices.
 
-Rollover relies on three things:
+<<indices-create-data-stream, Data streams>> are the recommended abstraction for managing time-series
+data as they automatically track the write index whilst also keeping the amount of configuration to a
+minimum.
+
+Rolling over using <<indices-create-data-stream, data streams>> only requires creating a
+<<indices-templates,composable index template>> containing a name or a wildcard pattern for the data stream
+in the `index_patterns` and a `data_stream` definiton that contains the `timestamp_field` property which
+will be present in the time-series documents that will be ingested.
```

Comment:
I think these requirements are incomplete as a mapping is now required for the timestamp field.
Either way, I think a bulleted list makes it more readable.

```suggestion
Each data stream requires a <<indices-templates,composable index template>> that contains:

* A name or wildcard (`*`) pattern for the data stream.

* The data stream's timestamp field. This field must be mapped as a
  <<date,`date`>> or <<date_nanos,`date_nanos`>> field datatype and must be
  included in every document indexed to the data stream.
  
  * The mappings and settings applied to each backing index when it's created.
```